### DISCUSSION TITLE
Ctrl-e will not change the color of zsh-autosuggestions.

### DISCUSSION THREAD

**Discussion by tshu-w** [marlonrichert/zsh-autocomplete#563] [Q&A] [answered]
After upgrading to macOS 13.3 (22E252), I encountered a strange conflict between zsh-autocomplete and zsh-autosuggestions. I am not sure if this is an issue with zsh-autocomplete, so I submitted a question.

## Environment
* Operating system: macOS
* Terminal emulator: Terminal.app & iTerm2
```zsh
apple darwin22.0 /bin/zsh /bin/zsh zsh-5.9-0-g73d3173
/Users/wangtianshu/.local/share/znap/zsh-snap/functions/.znap.source:26
/Users/wangtianshu/.local/share/znap/zsh-snap/functions/znap:42
/Users/wangtianshu/.config/zsh/.zshrc:23
5cc9da1 (grafted, HEAD -> main, origin/main, origin/HEAD) Fix typo in 8ef6b73
```

## Steps to reproduce
```zsh
cd $(mktemp -d)
git clone --depth 1 -- https://github.com/marlonrichert/zsh-autocomplete.git
git clone --depth 1 -- https://github.com/zsh-users/zsh-autosuggestions
> .zshrc <<EOF
PS1='%# ' PS2= RPS2='%^'; setopt transientrprompt interactivecomments
source ~/zsh-autosuggestions/zsh-autosuggestions.zsh
source ~/zsh-autocomplete/zsh-autocomplete.plugin.zsh
EOF
env -i HOME=$PWD PATH=$PATH TERM=$TERM ${TERMINFO:+TERMINFO=$TERMINFO} zsh -d
```
1. type `ls zsh-autosuggestions` and press `enter`
2. type `ls zsh` and press`Ctrl-e`
3. The color of `-autosuggestions` doesn't not change

<img width="1755" alt="Screenshot 2023-03-29 at 21 44 33" src="https://user-images.githubusercontent.com/13161779/228558084-d7b914b2-fab6-4503-8b23-39582e53de43.png">

The color change when not using zsh-autocomplete:
```zsh
```zsh
cd $(mktemp -d)
git clone --depth 1 -- https://github.com/zsh-users/zsh-autosuggestions
> .zshrc <<EOF
PS1='%# ' PS2= RPS2='%^'; setopt transientrprompt interactivecomments
source ~/zsh-autosuggestions/zsh-autosuggestions.zsh
EOF
env -i HOME=$PWD PATH=$PATH TERM=$TERM ${TERMINFO:+TERMINFO=$TERMINFO} zsh -d
```
<img width="1755" alt="Screenshot 2023-03-29 at 21 32 12" src="https://user-images.githubusercontent.com/13161779/228558355-290020f0-3d92-4d2c-8748-0454f4ce171f.png">

**Comment by marlonrichert** [comment]
I'm unable to reproduce this with the latest version of the code. Please update and try again.

**Reply by tshu-w** [reply]
Hi, I got  `.autocomplete.async.complete.fd-widget:3: not a directory: /Users/wangtianshu/.local/state/zsh-autocomplete/log/2023-04-06.log` after pull, Is there any mistakes?
<img width="2032" alt="Screenshot 2023-04-06 at 21 08 05" src="https://user-images.githubusercontent.com/13161779/230387867-993fae63-5931-488a-8335-1efa459e4130.png">

---
Update: it's caused by the last commit. I checkout 86ffb11 and everything works fine now.

**Reply by marlonrichert** [reply, marked as answer]
It's a known issue: #571.
